okio/js/src/main/kotlin/okio/Buffer.kt

```diff
@@ -15,4 +15,149 @@
  */
 package okio
 
-actual class Buffer : BufferedSource, BufferedSink
+actual class Buffer : BufferedSource, BufferedSink {
```

Comment:
FWIW I think Buffer fits better in common completely.

Reply:
Again, lots of java deps. Unless it's possible to have a non-`expects` common class enhanced by a platform specific one? Didn't think that was the case.

Reply:
That would be the use case for concrete implementations being allowed in `expect` classes, which is not there at the moment. For now we'll have to work around it in the same way we did with `ByteString`.